Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 55;
```

Comment:
Should we revert these unrelated changes?

Reply:
I actually think it's OK to land these changes. Otherwise, Xcode will keep making them whenever you modify this project.

Reply:
Looked it up, and I think this corresponds to Xcode 13.0, so yes, it should be fine.